### ISSUE TITLE
VB to C#: Nameof keyword causes exception

### ISSUE THREAD

**Issue by bfreezer99** [icsharpcode/CodeConverter#613]
### Steps to reproduce
1.  Create VB code with something similar to: Dim name = NameOf(Global.System.Collections.Generic.Dictionary(Of T, TU))
2.  Run codeconv

### Error message shown
```csharp
 /* Cannot convert LocalDeclarationStatementSyntax, System.InvalidCastException: Unable to cast object of type 'Microsoft.CodeAnalysis.CSharp.Syntax.EmptyStatementSyntax' to type 'Microsoft.CodeAnalysis.CSharp.Syntax.LocalFunctionStatementSyntax'.
               at System.Linq.Enumerable.<CastIterator>d__97`1.MoveNext()
               at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
               at ICSharpCode.CodeConverter.CSharp.MethodBodyExecutableStatementVisitor.<VisitLocalDeclarationStatement>d__31.MoveNext() in D:\GitWorkspace\CodeConverter\CodeConverter\CSharp\MethodBodyExecutableStatementVisitor.cs:line 109
            --- End of stack trace from previous location where exception was thrown ---
               at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
               at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
               at ICSharpCode.CodeConverter.CSharp.HoistedNodeStateVisitor.<AddLocalVariablesAsync>d__6.MoveNext() in D:\GitWorkspace\CodeConverter\CodeConverter\CSharp\HoistedNodeStateVisitor.cs:line 47
            --- End of stack trace from previous location where exception was thrown ---
               at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
               at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
               at ICSharpCode.CodeConverter.CSharp.CommentConvertingMethodBodyVisitor.<DefaultVisitInnerAsync>d__3.MoveNext() in D:\GitWorkspace\CodeConverter\CodeConverter\CSharp\CommentConvertingMethodBodyVisitor.cs:line 29

            Input:
                        Dim name = NameOf(Global.System.Collections.Generic.Dictionary(Of T, TU))

             */
```
### Details
using most recent command line version

**Comment by GrahamTheCoder** [comment]
I attempted to fix this for #592 on the vs extension by forcing a solution build since it can be caused by a missing dependency.
I'll try to repro with code conv, if you have an example which definitely builds but then consistently fails conversion, please zip and send it since I have had trouble reproducing this last time.
Thanks for the report!

**Comment by GrahamTheCoder** [comment]
Tracking in #638
